### DISCUSSION TITLE
Possible to have multiple evolu "spaces" on one domain?

### DISCUSSION THREAD

**Discussion by negue** [evoluhq/evolu#334] [General]
I'm thinking / trying out evolu if it fits my needs,

I often create static web apps (hosted on github for example) which means they have sometime the same subdomain

Can evolu handle different namespaces ? Or will both of my static apps use only one evolu datastore 

Couldn't find anything so far on Github

**Comment by steida** [comment]
Hi, Evolu can't handle different "namespaces" (different DBs under the same domain) because it still supports the LocalStorage for old Safari, but I think it's time to remove that limitation so that it will support that use case soon.

Related:
- https://github.com/evoluhq/evolu/issues/149
- https://github.com/evoluhq/evolu/issues/309

**Comment by steida** [comment]
Evolu already supports multiple instances on the same domain.
